CovertInfiltration/Src/CovertInfiltration/Classes/X2ActivityTemplate_Mission.uc

```diff
@@ -100,27 +100,22 @@ static function array<StateObjectReference> GenericInitializeMissionRewards (XCo
 static function StateObjectReference InitMissionReward (XComGameState NewGameState, XComGameState_Activity ActivityState, X2RewardTemplate RewardTemplate)
 {
 	local XComGameState_Reward RewardState;
+	local XComGameState_HeadquartersResistance ResHQ;
 
 	RewardState = RewardTemplate.CreateInstanceFromTemplate(NewGameState);
+	ResHQ = class'UIUtilities_Strategy'.static.GetResistanceHQ();
 	
 	// If this is a chain proxy reward, send the chain state reference into it instead of the region reference
 	if (RewardState.GetMyTemplateName() == 'Reward_ChainProxy')
 	{
-		RewardState.GenerateReward(NewGameState,, ActivityState.GetActivityChain().GetReference());
+		RewardState.GenerateReward(NewGameState, ResHQ.GetMissionResourceRewardScalar(RewardState), ActivityState.GetActivityChain().GetReference());
```

Comment:
For `Reward_ChainProxy`, there will never be any modifiers since it's not a resource reward